nox.py

```diff
@@ -317,9 +319,19 @@ def mark_if_necessary(requirement):
                     requirement.strip())
         return requirement
 
+    def is_ignored(requirement):
+        """Ignores certain libraries."""
+        for library in IGNORED_LIBRARIES:
+            if requirement.startswith(library):
+                return True
```

Comment:
Don't care either way but don't think it's  too clever, a bit more English-esque if anything.